### DISCUSSION TITLE
이전에 만든 썸네일 재사용하기

### DISCUSSION THREAD

**Discussion by da-in** [Yangjaecheon-Otter-Guardians/simple-thumbnail#112] [Ideas]
### 주제

Features

### Body

일단 notion에 back back backlog 로 적혀있어서 들고왔습니다.🤔

### 어떻게 생겼을지 상상해보면
- **쇼핑몰 사이트에서 최근 조회한 상품!** 을 떠올리시면 좋겠군요.
- 최근 1주동안, 혹은 최근 제작한 N개를 보여주는 형태는 어떨까요.
- 개발은 Local Storage나 기타 저장소에 스윽 저장하면 되지 않을까요.

### 고민
- UX/UI 측면에서 `쉽고 간편한 서비스` 의 취지에 반한다면 넣어서는 안될 것 같습니다.

언젠가 우리의 이 아이디어도 빛을 보면 좋겠군요.

### BDD TASK

- [ ] 사이트에 접속하면 > 최근에 만든 썸네일을 볼 수 있다.
- [ ] 과거에 만든 썸네일 기록을 클릭하면 > 해당 썸네일 설정값이 불러와진다.

**Comment by headring** [comment]
유저의 이전 정보를 기록하는 것 좋다고 생각합니다.

고려해볼 수 있는 것이 있다면....
1. 이전에 사용했던 거라면 한 사용자의 기록일까요? 아님 전체 사용자의 기록일까요?
한 사용자의 기록이면 localStorage로 커버가 될 것 같고, 전체 사용자라면 `firebase`연결? 또는 백엔드 기능이 조금 필요할 수 있겠네요.

2. 만약 글자까지 저장을 한다면 문자열 정보는 어떻게 저장하는 것이 좋을까요?

**Reply by da-in** [reply]
일단 간단한 서비스니까 한 사용자 정도로 생각했구요,

저장한다면 로컬스토리지에 모든 정보를 json 형태로 저장한후, 불러들여 파싱해서 사용하면 되지 않을까 생각했습니다!

투머치 기능일 것 같은 걱정은 있어서, 나중에 이미지 컴포넌트가 생겨서 위치 값이 자유로워지거나 등의 이유로, 똑같은 썸네일을 다시 만들기에 기능상 무리가 있을 경우 도입을 고민해도... 좋을 것 같습니다!
